src/main/java/com/example/demo/dto/user/UserResponseDto.java

```diff
@@ -0,0 +1,16 @@
+package com.example.demo.dto.user;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
```

Comment:
you can use Data here
```suggestion
@Data
```